### ISSUE TITLE
Add 'everyone'-like user for all authenticated users

### ISSUE THREAD

**Issue by gardient** [binwiederhier/ntfy#978]
:bulb: **Idea**
Have an `everyone`-like special user for all authenticated users in the ACL.
something like `all_authenticated`/`*auth`

This would allow those self-host to set a basic/default set of ACLs for "registered" users

for example it would be excelent to set something like the UnifiedPush topics to be readable only by people who are authenticated

```
ntfy access '*auth' 'up*' ro
```

:computer: **Target components**

ntfy server

**Comment by hansblaauw64** [comment]
Yeah, that would be great. Normal topics created by a validated user should be set read only for other users. Makes much more sense to me.

**Comment by kjetilho** [comment]
I think this makes most sense for self-hosted.  And in that case, a simple switch, "require authentication" would be a lot simpler to implement than to extend the semantics of the Permission type, and still solve 99% of the use-cases.

And happily, this is available:  `auth-default-access deny-all` will require authentication for all operations.  It can also be set to `write-only` to allow anonymous publication (spamming 😀 )
